### PR TITLE
Add parameters for fastlane specific env and ios app id

### PR DESCRIPTION
**Pull request by abarnto**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -66,23 +66,18 @@ inputs:
...
     required: true
-    default: false
   match-build-type:
     description: "The build type to use when building the app."
     required: true
-    default: false
   browserstack-upload:
     description: "Boolean to tell the Action to upload the .ipa to Browserstack App Live after the build."
     required: false
-    default: false
```
Comment (francescoben) [inline action.yml]: @abarnto why did you remove the default value from all these inputs?

Comment (abarnto) [reply action.yml]: @francescoben Because they're all required, so the user has to provide them anyway (except for `browserstack-upload`, its removal seems an oversight, so I should restore it). If you don't agree, I'm going to do a revert of them all.

Comment (francescoben) [reply action.yml]: ok @abarnto, so restore the default value only for `browserstack-upload`

**Comment by francescoben** [comment]
closes #17

**Comment by francescoben** [comment]
@abarnto please update the CHANGELOG file with a section for the new version (2.1.0)

**Review by francescoben — APPROVED** [review]
